### PR TITLE
improve api call actions logs

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
before this PR, all durations of certain API actions would have been logged. this PR unifies the logging via a single function which only logs if the duration is above a threshold

### CODE REVIEW

**Review by ssd04 — APPROVED** [review]

**Comment by bogdan-rosianu** [comment]
> Maybe you should check also in the `dblookupext` package if some logs can be removed or changed

Checked. No occurrence

**Review by miiu96 — APPROVED** [review]

**Review by gabi-vuls — DISMISSED** [review]
System test passed.

**Review by miiu96 — APPROVED** [review]

**Review by ssd04 — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed

File: api/shared/logging/logging.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	"time"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+)
+
+var log = logger.GetOrCreate("api/shared/logging")
+const thresholdMinAPICallDurationToLog = 200 * time.Millisecond
+
+// LogAPIActionDurationIfNeeded will log the duration of an action triggered by an API call if it's above a threshold
+func LogAPIActionDurationIfNeeded(startTime time.Time, action string) {
```
Comment (iulianpascalau) [inline api/shared/logging/logging.go:14]: no unit tests for this

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**